request.go

```diff
@@ -91,6 +93,12 @@ func (r *Request) SetBlockNumber(blockNumber *big.Int) *Request {
 	return r
 }
 
+func (r *Request) SetOverrides(overrides map[common.Address]gethclient.OverrideAccount) *Request {
```

Comment:
Add comment to indicate that `SetOverrides`  only work when no block hash set.

Reply:
added